### ISSUE TITLE
silent hill shattered memories is really slow

### ISSUE THREAD

**Issue by jasper1412** [hrydgard/ppsspp#10618]
What happens?
It was really really slow when playing it.

What *should* happen?
Like most commercial games on psp, it should run at a reasonable speed.

What hardware and operating system are you running PPSSPP on? GPU might matter if it's a graphical issue.

Android 5.1, Mali-T720

Can you please enlighten me why this game is too slow unlike other games on ppsspp, is it my hardware limitation? I hope this would be fixed soon.  Thank you.

**Comment by LunaMoo** [comment]
Performance might improve or degrade over time, but as long as it's not a huge negative change(not caused by fixing emulation of previously broken stuff which could explain it ~ like for example game slowing down when some effect shows up that in the past simply wasn't working), performance is not a valid issue.

 It's completely normal for different games to have different requirements even if they look opposite to how you imagine they should run(ugly games quite often are made ugly by heavy effects or simply small budget which means lack of optimizations). In case of PPSSPP, some games will run on practically any potato or toster, but others will only work decently on powerful gaming pc's, you should not expect to be able to play everything on a mobile as it's impossible. Sometimes more demanding games can be patched via cwcheat to reduce requirements by completely removing some effects, but it's rare, usually such patches can be found in the game's thread on the forums(for example Test Drive Unlimited which has a very ugly, resolution-killing effect that's also insanely heavy to emulate).


 In other words don't expect to "play Silent Hill games at reasonable speed like most other psp games" afaik they're running in 60 fps and involve some heavy effects, some of which aren't even emulated correctly yet which means if anything, those games might become even more demanding in the future. Typically frameskipping is recommended way to run 60 fps games better on mobiles, but SH has some effects which if I recall get's broken by it unless forced to skipping only even number of frames.

 Anyway, enough explaining ~ closing since nothing can be done about running speed of demanding games on low power devices, especially old ones that have no access to any new technology that could be used for improved performance in emulation(ie Vulkan).

**Comment by jasper1412** [comment]
@LunaMoo thank you very much :)
